frontend/src/components/Post.js

```diff
@@ -1,79 +1,67 @@
-import React from "react";
+import React, { useState, useEffect } from 'react'
 import axios from "axios";
 import { Link } from "react-router-dom";
 import { PostComment } from "./Comment/PostComment";
 import "./Post.scss";
 import '../config';
 import CommentList from "./Comment/CommentList";
+const Post = (props) => {
 
-class Post extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            Picture: undefined
-        }
-    }
-
-    componentDidMount() {
-        this.loadPicture();
-    }
+    const [Picture, setPicture] = useState(undefined);
+    const [refreshComment, setRefreshComment] = useState(false);
 
-    componentDidUpdate(prevProps, prevState, snapshot) {
-        if (prevProps.id !== this.props.id) {
-            this.loadPicture();
-        }
-    }
+    useEffect(() => {
+        loadPicture();
+    }, [props.id])
 
-    loadPicture() {
-        axios.get(global.config.BACKEND_URL + "/picture/" + this.props.id).then(
+    const loadPicture = () => {
```

Comment:
Can loadPicture be moved to insude the useEffect? I don't believe this functionally creates a difference or any real issue but it will prevent getting a warning.